.env.example

```
@@ -1,27 +1,30 @@
-JWT_COOKIE_SECURE=false
+JWT_SECRET=
+JWT_ACCESS_EXPIRATION=
+JWT_REFRESH_EXPIRATION=
+JWT_COOKIE_SECURE=
 
-KAKAO_CLIENT_ID=your-rest-api-key
-KAKAO_CLIENT_SECRET=your-client-secret
-KAKAO_REDIRECT_URI=http://localhost:8080/api/v1/auth/kakao/callback
-KAKAO_MAP_REST_API_KEY=your-kakao-rest-api-key
+FRONTEND_ORIGIN=
+FRONTEND_OAUTH_CALLBACK_URI=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FRONTEND_OAUTH_CALLBACK_URI key should go before the FRONTEND_ORIGIN key

(UnorderedKey)